release/cli.yaml

```
@@ -185,6 +189,26 @@ spec:
         - $(params.bundle-target-repo)
         image: quay.io/konflux-ci/appstudio-utils:latest
         name: copy-bundle
+      # To ease the transtion from the old quay org to the new quay org,
+      # also push the two images to the old quay org at. Rather than
```

It's a typo, but I don't want to redo the tests.

```suggestion
      # also push the two images to the old quay org. Rather than
```